flake.nix

```diff
@@ -55,11 +55,13 @@
                 pkgs.python3Packages.pytest
                 pkgs.python3Packages.pytest-datafiles
               ];
-              checkPhase = "
+              checkPhase = let
+                sample = "${./nixui/tests/sample}";
+              in ''
                 export NIX_STATE_DIR=/build
-                export NIX_PATH=nixpkgs=${pkgs.path}
+                export NIX_PATH=nixpkgs=${pkgs.path}:nixos-config=${sample}/configuration.nix
```

Comment:
I don't think we should change this environment variable since it prevents users from opening their own configurations with nix-gui unless they clone the repo.

Using `NIX_PATH` makes sense though since https://github.com/nix-gui/nix-gui/pull/47 saves to `~/.config/nixgui/configurations/<nixos-config>` and it isn't risky to use

Reply:
This is only for `checkPhase` and doesn't have any effect on runtime.

Reply:
oh right, misread